treeherder/model/management/commands/import_perf_data.py

```diff
@@ -0,0 +1,73 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+
+from django.core.management.base import BaseCommand, CommandError
+from optparse import make_option
+from treeherder.client import PerfherderClient
+from treeherder.model.derived.jobs import JobsModel
+
+import concurrent.futures
```

Comment:
Hmm, it seemed to be installed already on my vagrant instance, maybe something else is already pulling it in. Still, we should add it explicitly I suppose, will do.